stdlib/optparse.pyi

```diff
@@ -27,8 +29,8 @@ NO_DEFAULT: tuple[str, ...]
 SUPPRESS_HELP: str
 SUPPRESS_USAGE: str
 
-def check_builtin(option: Option, opt, value: str): ...
-def check_choice(option: Option, opt, value: str) -> str: ...
+def check_builtin(option: Option, opt: str, value: str) -> complex: ...
```

Comment:
This can return complex, float, or int. I guess that does indeed simplify to `complex` but I think it's worth adding a comment.
```suggestion
# Can return complex, float, or int depending on the option's type
def check_builtin(option: Option, opt: str, value: str) -> complex: ...
```